### DISCUSSION TITLE
Connection refused

### DISCUSSION THREAD

**Discussion by OutbreakMalwareThreat** [community/community#146339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

                
Error downloading from github.com
2507998 bytes downloaded from cloud.githubusercontent.com at 3.62 Mbps
2507998 bytes downloaded from avatars.githubusercontent.com at 36.80 Mbps
2507998 bytes downloaded from github.githubassets.com at 6.51 Mbps
2507998 bytes downloaded from australiaeast.github-debug.com at 2.55 Mbps
2507998 bytes downloaded from brazilsouth.github-debug.com at 1.15 Mbps
2507998 bytes downloaded from centralindia.github-debug.com at 5.35 Mbps
2507998 bytes downloaded from fra.github-debug.com at 2.44 Mbps
2507998 bytes downloaded from iad.github-debug.com at 1.59 Mbps
2507998 bytes downloaded from japaneast.github-debug.com at 2.82 Mbps
2507998 bytes downloaded from koreacentral.github-debug.com at 3.16 Mbps
Error downloading from sea.github-debug.com
2507998 bytes downloaded from southafricanorth.github-debug.com at 2.26 Mbps
2507998 bytes downloaded from southeastasia.github-debug.com at 4.12 Mbps
2507998 bytes downloaded from github-cloud.s3.amazonaws.com at 0.35 Mbps

                
106.216.226.226

                
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

                
90df465fa921bd74480c2fc78ee83ec9

                
C2C9:28963B:7498:12593E:67519CC3

                
                  

                    
traceroute to 106.216.226.226 (106.216.226.226), 30 hops max, 60 byte packets
 1  *
 2  *
 3  *
 4  *
 5  *
 6  *
 7  *
 8  *
 9  *
10  *
11  *
12  *
13  *
14  *
15  *
16  *
17  *
18  *
19  *
20  *
21  *
22  *
23  *
24  *
25  *
26  *
27  *
28  *
29  *
30  *

                    
PING 106.216.226.226 (106.216.226.226) 56(84) bytes of data.

--- 106.216.226.226 ping statistics ---
3 packets transmitted, 0 received, 100% packet loss, time 2026ms

                    
Start: 2024-12-05T04:31:15-0800
HOST: 463f7d3fa081dc9614eddd68719ebb49               Loss%   Snt   Last   Avg  Best  Wrst StDev

                    
;; communications error to 10.6.96.5#53: timed out
Host 226.226.216.106.in-addr.arpa. not found: 3(NXDOMAIN)

**Comment by OutbreakMalwareThreat** [comment]
what do i do i am getting a connection refused

**Comment by jormunrod** [comment]
It looks like you're experiencing issues with downloading from various GitHub servers. Connection errors can be caused by several factors, such as local network issues, firewall blocks, or problems on GitHub's servers.

Here are a few steps you can try to resolve the issue:

1. **Check your internet connection**: Ensure your internet connection is stable. You might want to restart your router or try using a different network to see if the issue persists.

2. **Review firewall and proxy settings**: If you're using a firewall or proxy, check to see if it's blocking access to GitHub servers or certain domains like cloud.githubusercontent.com.

3. **Try a different traceroute**: It looks like the traceroute you ran didn't provide useful information. You could try running a traceroute to other GitHub servers (for example, github.com) to see if any part of the network is causing the issue.

4. **Check GitHub status**: Sometimes GitHub may experience outages or issues on their end. You can check the [GitHub Status page](https://www.githubstatus.com/) to see if there are any ongoing incidents.

5. **Try another browser or Git client**: If you're using a browser, try switching to a different one. If you're using Git through the command line, make sure your Git client is up-to-date and configured correctly.

If the problem persists after trying these steps, it would be helpful to know more details about your setup (e.g., operating system, Git client, or browser) to troubleshoot further.

I hope this helps!

**Reply by mecodeatlas** [reply]
Hey @ShubhMisra! Did @jormunrod response help you with your question? :)

**Reply by OutbreakMalwareThreat** [reply]
i think my ip is blocked

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @ShubhMisra!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
